js/app/controllers/calcontroller.js

```diff
@@ -140,8 +140,10 @@ app.controller('CalController', ['$scope', 'Calendar', 'CalendarService', 'VEven
 				$scope.$apply();
 			});
 		} else {
-			$scope.calendarsPromise = CalendarService.getPublicCalendar(constants.publicSharingToken).then(function(calendar) {
-				$scope.calendars = [calendar];
+			constants.publicSharingToken.split(".").forEach( (token) => {
+				$scope.calendarsPromise = CalendarService.getPublicCalendar(token).then(function(calendar) {
+					$scope.calendars = [calendar];
```

Comment:
This is still wrong. You are always overwriting `$scope.calendars` with a new array with one item here.
Instead you need to initialize an empty array first and push each calendar into the array individually.